stable/pomerium/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "pomerium.fullname" . }}
+{{ if .Values.serviceMonitor.namespace -}}
+  namespace: {{ .Values.serviceMonitor.namespace }}
+{{- end }}
+  labels:
+    helm.sh/chart: {{ template "pomerium.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
Shouldn't there be also a `app.kubernetes.io/name: {{ template "pomerium.name" . }}` here (and the same for the selector)? That's what was done for the `Deployment`s at least

Reply:
It seemed repetitive when rendered out.  Harmless to retain, but it doesn't do much.

The deployments have `app.kubernetes.io/name` with the individual service names, so I left the selector intentionally loose to capture all of the deployments the chart installs with one rule.

Reply:
I better understand, I didn't get that the selector was for the deployments :)

Reply:
👍